src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -796,6 +799,7 @@ public static ImmutablePair<Server, ServletContext> _createWebServer(String cont
         context.setConfigurations(new Configuration[]{new WebXmlConfiguration()});
         context.addBean(new NoListenerConfiguration(context));
         server.setHandler(context);
+        JettyWebSocketServletContainerInitializer.configure(context, null);
```

Comment:
@basil FYI this JTH release breaks a WebSocket test in the `kubernetes` plugin:

```
java.lang.NullPointerException: Cannot invoke "org.eclipse.jetty.websocket.server.JettyWebSocketServerContainer.upgrade(org.eclipse.jetty.websocket.server.JettyWebSocketCreator, javax.servlet.http.HttpServletRequest, javax.servlet.http.HttpServletResponse)" because the return value of "org.eclipse.jetty.websocket.server.JettyWebSocketServerContainer.getContainer(javax.servlet.ServletContext)" is null
	at jenkins.websocket.Jetty10Provider.handle(Jetty10Provider.java:61)
	at jenkins.websocket.WebSockets.lambda$upgrade$0(WebSockets.java:70)
```

I am still trying to understand why; in the meantime, https://github.com/jenkinsci/kubernetes-plugin/pull/1275/commits/dcc8137937248b52ff7754287ccb777e1f503d9a

Reply:
Ah, never mind, I think this is due to a stale copy in that plugin (https://github.com/jenkinsci/jenkins-test-harness/pull/289#issuecomment-809560892).

Reply:
Glad to hear it, especially given that I tested this case by running the Swarm plugin's `JenkinsRule`-based tests (which use WebSocket) against this PR prior to its merge and release.